velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -515,6 +515,40 @@ struct DateFromUnixDateFunction {
   }
 };
 
+/// Truncates a timestamp to a specified time unit. Return null if the
+/// format is invalid, microseconds and abbreviated unit string are allowed.
```

Comment:
Return NULL if the format is invalid. Format as abbreviated unit string and "microseconds" are allowed.